doc/devel/document.rst

```diff
@@ -1045,6 +1045,104 @@ subdirectory, but  :file:`galleries/users_explain/artists` has a mix of
 any ``*.rst`` files to a ``:toctree:``, either in the ``README.txt`` or in a
 manual ``index.rst``.
 
+
+.. _writing-user-guide:
+
+Write user guide documentation
+==============================
+
+The pages in the :ref:`users-guide-index` are a mix of :ref:`ReST <writing-rest-pages>`
+and :ref:`sphinx gallery <writing-examples-and-tutorials>` pages and should follow the
+respective formatting conventions. The goal of the user guide is to explain how
+Matplotlib works to someone who is unfamiliar with the library. To maintain the
+consistency and cohesiveness of the user guide, documentation that is contributed to
+this section should conform to the following guidelines:
+
+Purpose: Explain how Matplotlib works
+-------------------------------------
+
+The aim of the user guide is to teach the conceptual abstractions on which the
+Matplotlib API is developed so that users can understand how to fit the individual
+components of the library together. Therefore, content should be concept oriented
+rather than focused on specific tasks, e.g.
+
+  Lines in Matplotlib
+
+rather than *How do I make a squiggly yellow line?*
+
+Audience: new users
+-------------------
+
+The audience for the user guide are readers who are getting introduced to using Matplotlib
+through the guide; therefore, the user guide should be written with the assumption that
+the reader does not yet know what Matplotlib calls a given visualization task nor
+how any task is accomplished in Matplotlib. For example, each document should first
+introduce or define the object/module/concept that it is discussing and why it is
+important for the reader. e.g.
+
+  The `~.Line2D` class is an abstraction of a line and the ``LineCollection`` class is
+  an abstraction of a set of lines. ``Line2D`` and ``LineCollection`` objects manage
+  the properties and behavior of almost every line in an image. This means that one way
+  to modify a line is to call methods on its underlying object.*
+
+As the user guide is intended to be read in a somewhat linear fashion, it should be
+written with the assumption that the reader has read previous sections and cross
+reference to those sections when it is critical that the reader has read those sections.
+
+
+Scope: Visualization using Matplotlib
+-------------------------------------
+
+Many concepts in Matplotlib assume a grounding in visualization, statistics, and
+other topics to understand how they work. These concepts should be
+contextualized using common terminilogy, but the focus should not stray from the
+Matplotlib topic, e.g.
+
+  ``Line2D`` objects take as input either pairwise coordinates (x,y) or y with an implicit
```

Comment:
I find this example very confusing, "pairewise coordinates" usually means `[(x1, y1), (x2, y2), ...]` (which we will take and plot, but treat both columns as y.